### PR TITLE
Greater icons size on mobile and tablet

### PR DESCRIPTION
**Pull request by pgiraud**

### CODE REVIEW

**Comment by ger-benjamin** [comment]
That seems right. but I've not tested...
For me, you can merge or add an example if you have any doubt.

**Comment by pgiraud** [comment]
You can have a look here:
https://pgiraud.github.io/ngeo/mobile_icons_size/examples/contribs/gmf/apps/mobile/

File: contribs/gmf/less/mobile.less [hunk 1]
```diff
@@ -227,15 +227,24 @@ gmf-map {
...
 
 .ol-control button,
-.ol-touch .ol-control button {
-  font-size: 1em;
+.ol-touch .ol-control button,
+.fa:after,
+.fa:before,
+.gmf-icon:after,
+.gmf-icon:before {
+  font-size: 2.4rem;
```
Comment (ybolognini) [inline contribs/gmf/less/mobile.less]: I feel icons "data", "tools" and "geolocation" are too big, in comparison with font size of menu options. I would suggest: 

`font-size: 2.0rem`

and 

`button {
   width: 2.1em;
   height: 2.1em;
 }`

Comment (pgiraud) [inline contribs/gmf/less/mobile.less]: Let's go with this first and adjust later.
Can I merge?

Comment (ybolognini) [inline contribs/gmf/less/mobile.less]: OK to try with my values?  If yes, then change and merge

Comment (pgiraud) [inline contribs/gmf/less/mobile.less]: My proposal:
screenshot from 2016-02-18 17 53 04
Yours:
screenshot from 2016-02-18 17 53 12

I agree, it looks better.

**Comment by pgiraud** [comment]
Will merge when travis is happy.

**Pull request merged**